public/js/spa.js

```diff
@@ -408,8 +362,36 @@ $(document).on('loaded', function() {
             var modelsName = plural(modelName);
             var viewPathPrefix = "";
 
+            var modelView = function() {
+              console.log('thing:', makiConfig.config);
+              console.log('resource:', Resources[modelName]);
+
+              var resource = Resources[modelName];
+              // TODO: use .get when single, .query when multiple
+              var template = resource.templates.query;
+
+              // TODO: auto-populate this
+              var locals = {
+                page: {},
+                messages: {
+                  info: [],
+                  warning: [],
+                  success: [],
+                  error: []
+                },
+                user: {},
+                config: makiConfig.config
+              };
+
+              // TODO: use real data!
+              //locals[ resource.names.query ] = $scope.items;
```

Comment:
`$scope` is being called here in the context of `initDefaultRoutes`, which runs inside of `initApp`.  Is there a specific injection we're missing to access the scope at this point?